roles/openldap-server/tasks/main.yml

```diff
@@ -106,7 +106,7 @@
     lineinfile:
       path: /etc/default/slapd
       regexp: '^SLAPD_SERVICES='
-      line: SLAPD_SERVICES="ldap://127.0.0.1/ ldaps:///"
+      line: SLAPD_SERVICES="ldap://{{ openldap_server }}/ ldaps:///"
```

Comment:
we should not expose the ldap server (should stay on localhost - like the ipv6 instance)